test/unit/config/index.js

```diff
@@ -174,6 +174,73 @@ describe('Config', () => {
 
                 expect(() => new Config('./docpub.config')).to.throw(/string/);
             });
+
+            describe('renderer', () => {
+                it('should not throw if `renderer` is not provided', () => {
+                    stubConfig_({
+                        path: './docpub.config',
+                        config: {
+                            username: 'default_username',
+                            token: 'default_token',
+                            url: 'default_url'
+                        }
+                    });
+
+                    expect(() => new Config('./docpub.config')).to.not.throw();
+                });
+
+                it('should throw if `highlight` is not a boolean', () => {
```

Comment:
Can we please check that overloaded boolean option work? I worry about type conversions - maybe args or CLI variables would be passed as strings.
If so, probably we need to add `parseEnv` and `parseCli` as `JSON.parse` to option configuration.